deploy-stg.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+timestamp=$(date +%s)
+
+# remember initial branch
+dev_branch=$(git rev-parse --abbrev-ref HEAD)
+
+# replace possible "/" with "-"
+dev_branch_sanitized=${dev_branch/\//-}
+
+stg_branch="stg-v0.0.${timestamp}-${dev_branch_sanitized}"
+
+git checkout -b "$stg_branch"
+git push origin "$stg_branch"
```

Comment:
Consider doing:

```
git push origin HEAD:"$stg_branch"
```

Then you could simplify the script by removing the git checkouts because a local branch is not needed then.

Reply:
Oh, I didn't know that!